rewrite-java/src/main/java/org/openrewrite/java/format/NormalizeLineBreaksVisitor.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.format;
+
+import org.openrewrite.Tree;
+import org.openrewrite.internal.ListUtils;
+import org.openrewrite.internal.lang.Nullable;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavadocVisitor;
+import org.openrewrite.java.tree.*;
+import org.openrewrite.style.FormatStyle;
+
+public class NormalizeLineBreaksVisitor<P> extends JavaIsoVisitor<P> {
+    @Nullable
+    private final Tree stopAfter;
+
+    private final FormatStyle style;
+
+    public NormalizeLineBreaksVisitor(FormatStyle style) {
+        this(style, null);
+    }
+
+    public NormalizeLineBreaksVisitor(FormatStyle style, @Nullable Tree stopAfter) {
+        this.style = style;
+        this.stopAfter = stopAfter;
+    }
+
+    @Override
+    public Space visitSpace(Space space, Space.Location loc, P p) {
+        Space s = space.withWhitespace(normalizeNewLines(space.getWhitespace(), style.getUseCRLFNewLines()));
+
+        return s.withComments(ListUtils.map(s.getComments(), comment -> {
+            Comment c = comment;
+            if (c.isMultiline()) {
+                if (c instanceof Javadoc) {
+                    c = c.withSuffix(normalizeNewLines(c.getSuffix(), style.getUseCRLFNewLines()));
+                    return (Comment) new JavadocVisitor<Integer>() {
```

Comment:
You could re-use the JavadocVisitor. no state is involved, so you could make a single, static instance to reduce garbage collection pressure.